e2e/integration/solid-vinxi/src/lib/auth.ts

```diff
@@ -11,6 +11,11 @@ export const auth = betterAuth({
 	emailAndPassword: {
 		enabled: true,
 	},
+	trustedOrigins: [
+		baseURL,
```

Comment:
I seem to remember that `baseURL` is automatically added to the trusted origins list.